woodwork/statistics_utils/_determine_best_outlier_method.py

```diff
@@ -0,0 +1,63 @@
+from collections import namedtuple
+
+import numpy as np
+import pandas as pd
+from statsmodels.stats.stattools import medcouple
+
+import woodwork as ww
+
+
+def _sample_for_medcouple(series):
+    """Samples the data in order to calculat the medcouple statistic.
+
+    Args:
+        series (Series): Data on which sampling will occur.
+
+    Note:
+        The calculation of the medcouple statistic has a large memory requirement of O(N**2), therefore a series over 10,000
+        observations will be randomly sampled up to 10,000.
+
+    Returns:
+        float: Medcouple statistic
+    """
+    np.random.seed(42)
```

Comment:
A constant seed will make sure that no matter how many times the random sampling is run on a series, the same observations will be selected every time.

Reply:
Do you want to make it an optional argument on the function?

Reply:
Also, another way to guarantee the same results is to use the first 10K rather than a random sample of 10K. Up to you.

Reply:
Depending on how the data has been sorted, the first X observations might not capture the distribution in any meaningful way, a random sample would be more robust. But I like the idea of adding an optional argument!